### PR TITLE
feat: tableflip example

### PR DESCRIPTION
**Pull request by linbuxiao**
#162

### CODE REVIEW

**Comment by ReneWerner87** [comment]
thx

- [ ] please add a readme with description about this example
- [ ] please add a link in the readme in our root directory
- [ ] please add a little bit more inline documentation
- [ ] optional - add tests

File: tableflip/go.mod [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module tableflip
+
+go 1.17
+
+require (
+	github.com/cloudflare/tableflip v1.2.3
+	github.com/gofiber/fiber v1.14.6
```
Comment (ReneWerner87) [inline tableflip/go.mod:7]: why do you need the old fiber version here?

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]

File: tableflip/go.mod (hunk 1 above)
Comment (linbuxiao) [reply tableflip/go.mod:7]: that's a mistake. I will fix it soon.

**Comment by linbuxiao** [comment]
@ReneWerner87 done. need review again.

**Comment by ReneWerner87** [comment]
image

can you maybe put some information about the word tableflip somewhere so that people understand that it is a graceful shutdown?

maybe next to the link in the root readme and more description under the readme in the example folder

even i did not know what to do with this word 

unfortunately an example is only useful if it is used, no user will click on this example if he does not immediately see what it is about

**Comment by linbuxiao** [comment]
> image
> 
> can you maybe put some information about the word tableflip somewhere so that people understand that it is a graceful shutdown?
> 
> maybe next to the link in the root readme and more description under the readme in the example folder
> 
> even i did not know what to do with this word
> 
> unfortunately an example is only useful if it is used, no user will click on this example if he does not immediately see what it is about

I understand what you mean. I will supplement it.

**Comment by linbuxiao** [comment]
I just add link to the repo and quoted its introduction. I don't know if that makes the example any clearer?

**Review by ReneWerner87 — APPROVED** [review]

**Comment by ReneWerner87** [comment]
> Your suggestion makes this example more helpful to others, thank you.
> 
> I added link to the repo and quoted its introduction. I don't know if that makes the example any clearer?

now its okay

**Pull request merged**